pages/app/presenters/refinery/pages/section_presenter.rb

```diff
@@ -69,10 +69,15 @@ def wrap_content_in_tag(content)
       end
 
       def sanitize_content(input)
-        output = sanitize(input,
-          tags: Refinery::Pages::whitelist_elements,
-          attributes: Refinery::Pages::whitelist_attributes
-        )
+        output =
+            if Refinery::Core.regex_white_list
```

Comment:
For me `whitelist_regex` should be a config and not a boolean.

No need to make and if else statement, we should keep the CustomScrubber only.